### PR TITLE
Np 47570 add more correction lists

### PR DESCRIPTION
**Pull request by elkebab**
# Description

Link to Jira issue: https://sikt.atlassian.net/browse/NP-47570

Legger til to nye rettelister:

bilde

# Checklist

Ensure that the changes are aligned with the expectations of PO/designer before marking the PR as ready for review.

Also ensure that the following criterias are met:

- [x] The changes are working as expected
- [x] The changes are tested OK for different screen sizes
- [x] The changes are tested OK for a11y
- [x] Interactive elements have data-testids
- [x] I have done a QA of my own changes

_Note: some of these criterias may not always be relevant, and can simply be marked as completed._

### CODE REVIEW

File: src/pages/messages/components/NviCorrectionList.tsx [hunk 1]
```diff
@@ -36,6 +39,21 @@ const correctionListConfig: CorrectionListSearchConfig = {
       scientificValue: [ScientificValueLevels.LevelOne, ScientificValueLevels.LevelTwo].join(','),
     },
   },
+  AntologyWithoutChapter: {
+    i18nKey: 'tasks.nvi.correction_list_type.antology_without_chapter',
+    queryParams: {
+      categoryShould: [BookType.Anthology],
+      hasNoChildren: false,
+    },
+    // TODO: disable category filter
```
Comment (elkebab) [inline src/pages/messages/components/NviCorrectionList.tsx:48]: Avhenger av https://github.com/BIBSYSDEV/NVA-Frontend/pull/6232

File: src/api/searchApi.ts [hunk 2]
```diff
@@ -338,6 +339,7 @@ export enum ResultParam {
   Order = 'order',
   Project = 'project',
   PublicationLanguageShould = 'publicationLanguageShould',
+  PublicationPages = 'publicationBookPages',
```
Comment (elkebab) [inline src/api/searchApi.ts:342]: Har en diskusjon med Stig om denne skal hete `publicationBookPages` eller `publicationPages`, så kan være det blir en bitteliten endring her

Comment (elkebab) [reply src/api/searchApi.ts]: Det ble `publicationPages`!

**Review by iBuzza — DISMISSED** [review]
💹

File: src/components/CategorySelector.tsx [hunk 3]
```diff
@@ -188,17 +188,18 @@ const RegistrationTypesRow = ({ mainType, registrationTypes, onChangeType }: Reg
...
+export const CategoryChip = ({ category, onClickChip, disabled = !!category.disableText }: CategoryChipProps) => {
   const { t } = useTranslation();
 
   return (
     <Tooltip title={category.disableText}>
       <span>
         <Chip
           data-testid={dataTestId.registrationWizard.resourceType.resourceTypeChip(category.value)}
-          disabled={!!category.disableText}
+          disabled={disabled || !!category.disableText}
```
Comment (elkebab) [inline src/components/CategorySelector.tsx:202]: kan forenkles til å bare bruke disabled?

**Pull request merged**